### PR TITLE
remove deprecated UserMap#getUserFromBukkit usage

### PR DESCRIPTION
**Pull request by uf0h**
spotted unnecessary deprecated usage, replace UserMap#getUserFromBukkit with #getPlayer (with offline bool true).

Tested cases:
online player
offline player
offline player which has never joined

### CODE REVIEW

**Review by pop4959 — COMMENTED** [review]
Does this even need to be run asynchronously anymore? Would be good to look into, because the code in UserMap for the method you removed (as shown below) implies that this may be the only reason that it was. In that case, it would be good to simplify the code further.

https://github.com/EssentialsX/Essentials/blob/c86e1a5973445c72c31e68271acd554954b5d6b7/Essentials/src/com/earth2me/essentials/UserMap.java#L239

**Comment by uf0h** [comment]
> 
> 
> Does this even need to be run asynchronously anymore? Would be good to look into, because the code in UserMap for the method you removed (as shown below) implies that this may be the only reason that it was. In that case, it would be good to simplify the code further.
> 
> https://github.com/EssentialsX/Essentials/blob/c86e1a5973445c72c31e68271acd554954b5d6b7/Essentials/src/com/earth2me/essentials/UserMap.java#L239

You're right, it doesn't. Fixed it up :) 4de450c

**Review by pop4959 — DISMISSED** [review]

File: Essentials/src/com/earth2me/essentials/commands/Commandseen.java [hunk 1]
```diff
@@ -34,55 +34,36 @@ protected void run(final Server server, final CommandSource sender, final String
...
             player = ess.getUser(uuid);
-        } catch (IllegalArgumentException ignored) { // Thrown if invalid UUID from string, check by name.
-            player = ess.getOfflineUser(args[0]);
-        }
-
-        if (player == null) {
-            if (sender.isAuthorized("essentials.seen.ipsearch", ess) && FormatUtil.validIP(args[0])) {
+        } catch (IllegalArgumentException ignored1) {
+            // check by ip
+            if (FormatUtil.validIP(args[0])) {
```
Comment (pop4959) [inline Essentials/src/com/earth2me/essentials/commands/Commandseen.java:45]: You removed a permission check here.

**Review by pop4959 — CHANGES_REQUESTED** [review]

**Comment by JRoy** [comment]
Closing due to staleness + probable side effects

**Pull request closed without merging**